src/Symfony/Component/Form/Extension/Core/ChoiceList/ChoiceListInterface.php

```diff
@@ -25,24 +25,14 @@
  *              in the HTML "value" attribute.
  *
  * @author Bernhard Schussek <bschussek@gmail.com>
+ *
+ * @deprecated Deprecated since Symfony 2.7, to be removed in Symfony 3.0.
+ *             Use {@link \Symfony\Component\Form\ChoiceList\ChoiceListInterface}
+ *             instead.
  */
-interface ChoiceListInterface
+interface ChoiceListInterface extends \Symfony\Component\Form\ChoiceList\ChoiceListInterface
```

Comment:
I don't see any call to `trigger_error()` in this patch, so deprecation notices are not in place as far as I can tell. Am I missing something here?